crd/overlayextensionconfig/api/v1alpha1/overlayextensionconfig_types.go

```diff
@@ -34,11 +34,14 @@ type OverlayExtensionConfigList struct {
 // +kubebuilder:validation:XValidation:rule="!has(oldSelf.extensionIPRange) || has(self.extensionIPRange)", message="ExtensionIPRange is required once set"
 type OverlayExtensionConfigSpec struct {
 	// ExtensionIPRange field defines a CIDR that should be able to reach routing domain ip addresses.
-	// +kubebuilder:validation:Optional
```

Comment:
do we want to make it a required field?

Reply:
Do not need to call it as required

Reply:
we probably shouldn't mark it required, it'll break all the existing clusters which are not using it

Reply:
Left this as optional

Reply:
Matt is suggesting making the new field optional to not break existing clusters which are not populating this field currently.

We will validate it as required in OEC reconciller for new clusters to always require this field.